project/Build.scala

```diff
@@ -47,8 +47,10 @@ object ExposedValues extends AutoPlugin {
 
 object Build {
 
-  val isGeneratingEclipse =
-    Properties.envOrElse("GENERATING_ECLIPSE", "false").toBoolean
+  val isGeneratingEclipse = {
```

Comment:
Hum, maybe this `val` should be renamed?

Reply:
Done.